src/main/resources/init_db.sql

```diff
@@ -1,9 +1,17 @@
-CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
-USE `taxi`;
+CREATE DATABASE `taxi_service` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
 
 CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(225) DEFAULT NULL,
+                                 `country` varchar(225) DEFAULT NULL,
+                                 `is_deleted` tinyint DEFAULT '0',
+                                 PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=18 DEFAULT CHARSET=utf8mb3;
+
+CREATE TABLE `drivers` (
+                           `id` bigint NOT NULL AUTO_INCREMENT,
+                           `name` varchar(225) DEFAULT NULL,
+                           `license_number` varchar(225) DEFAULT NULL,
+                           `is_deleted` tinyint NOT NULL DEFAULT '0',
+                           PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=9 DEFAULT CHARSET=utf8mb3;
```

Comment:
the same as above

Reply:
all lines in init_db.sql is copy of Create Statement from mySQL

Reply:
I understand but if you use this script your AUTO_INCREMENT will start not from 1 but 9 that is wrong, so remove AUTO_INCREMENT